src/main/java/org/opensearch/sdk/handlers/OpensearchRequestHandler.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.sdk.handlers;
+
+import org.opensearch.extensions.OpenSearchRequest;
+import org.opensearch.sdk.NamedWriteableRegistryAPI;
+import org.opensearch.sdk.ExtensionsRunner;
+import org.opensearch.transport.TransportResponse;
+
+/**
+ * This class handles the request from OpenSearch to a {@link ExtensionsRunner#startTransportService(TransportService transportService)} call.
+ */
+
+public class OpensearchRequestHandler {
+    private NamedWriteableRegistryAPI namedWriteableRegistryApi = new NamedWriteableRegistryAPI();
```

Comment:
Let's rename it to `NamedWriteableRegistryAPI` -> `NamedWriteableRegistry`

Reply:
Do you means rename the `NamedWriteableRegistryAPI.java` to `NamedWriteableRegistry.java`

Reply:
All the places where we have `NamedWriteableRegistryAPI` -> `NamedWriteableRegistry`.

Reply:
There is another class called `NamedWriteableRegistry` get imported in `NamedWriteableRegistryAPI.java`. If we rename `NamedWriteableRegistryAPI` ->`NamedWriteableRegistry`, it will cause error. Can we use another name to avoid this error.

Reply:
@dbwiddis I'm pretty sure you can suggest a better name here. 
![actual-programming-vs-debating-30-minutes-on-how-to-name-a-variable-drake-meme](https://user-images.githubusercontent.com/16099877/193127979-dcb5e769-af1d-46db-95a5-0b27cf54cb81.jpeg)

Reply:
How about `ExtensionNamedWriteableRegistry`? :-)

![image](https://user-images.githubusercontent.com/9291703/193133072-f3e19d37-e6c9-40d9-8af7-c9b7bc42ec19.png)